.kokoro/release/common.cfg

```diff
@@ -47,3 +47,6 @@ before_action {
     }
   }
 }
+
+# Downloads docfx doclet resource. This will be in ${KOKORO_GFILE_DIR}/docfx-doclet-1.0-SNAPSHOT-jar-with-dependencies-143274.jar
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/docfx/docfx-doclet-1.0-SNAPSHOT-jar-with-dependencies-143274.jar"
```

Comment:
You might not need to be this explicit here - presume we'll update the file from time to time, why change several places?